modules/install/pages/getting-started-docker.adoc

```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
+
+. Install two instances of the official Couchbase Server container image.
++
+Make sure to run each of the following commands:
++
+[source,console]
+----
+$ docker run -d --name db1 -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase
+----
++
+[source,console]
+----
+$ docker run -d --name db2 -p 9091-9096:8091-8096 -p 12210-12211:11210-11211 couchbase
+----
++
+After running the above commands,
+three instances (`db1` and `db2`) of the latest https://hub.docker.com/_/couchbase/[official Couchbase Server container image^] are downloaded
+and installed onto the host computer.
+If a traditional installation of Couchbase Server is running locally on the host computer,
+the port mappings specified using the `-p` option may fail.
+Ensure that you stop any local instance of Couchbase Server before running these commands.
++
+NOTE: If you're using encrypted communication for the Couchbase Web Console, client, and server,
+and using XDCR, you need to open up additional ports.
+For details, refer to xref:install-ports.adoc[].
+
+. You can check the Docker logs to verify that each container has started:
++
+[source,console]
+----
+$ docker logs db1
+----
++
+[source,console]
+----
+$ docker logs db2
+----
++
+If the containers have started successfully, then each one will return the following output:
++
+[source,console]
+----
+Starting Couchbase Server -- Web UI available at http://<ip>:8091
+...
+----
++
+. Discover the local IP addresses of `db1` and `db2`.
++
+
+[source,console]
+----
+$ docker inspect --format '{{ .NetworkSettings.IPAddress }}' db1
+----
++
+[source,console]
+----
+$ docker inspect --format '{{ .NetworkSettings.IPAddress }}' db2
+----
++
+Note down the IP addresses as these will be needed for configuring the server nodes.
++
+Each instance is a Couchbase cluster, so you will need to access the UI for each cluster to add a server node.
+
++
+.Accessing the server UI
+|===
+|Instance  |Address
+
+|db1
+|http://localhost:8091
+
+|db2
+|http://localhost:9091
+|===
++
+. From a web browser, go to `+http://localhost:8091+` to access the Couchbase Web Console.
++
+If `db1` is up and running, you should see the Couchbase Server setup screen:
++
+image::welcome.png["The Couchbase Server setup screen.",400]
+. Click btn:[Setup New Cluster] and proceed through the setup wizard as normal.
++
+Refer to xref:manage:manage-nodes/create-cluster.adoc[] for instructions on using the setup wizard.
+You may need to lower the RAM allocation for various services to fit within the bounds of the container's resources.
++
+. From a web browser, go to `+http://localhost:9091+` to access the Couchbase Web Console for `db2`.
+
++
+image::welcome-join-existing-cluster.png["The Couchbase Server setup screen.",400]
+. Click btn:[Join Existing Cluster].
+
++
+image::join-cluster-details.png[Joining an existing cluster, 450]
++
+. Fill in the IP address associated with container `db1`, along with the administrator password.
++
+. Press btn:[Join Cluster]
```

Comment:
I though this section was about running multiple single-node clusters, so shouldn't be any discussion around adding nodes and rebalance.